### PR TITLE
CP: Update Vaadin and collaboration-engine

### PR DESCRIPTION
**Pull request by tulioag**
cherry-pick: #51

* Revert "Fix memory leak (#48)"
This reverts commit 5fb94e6d542e38936afd6fef496614aebf9ebec6.

* Updated versions in pom
Use vaadin 21 stable
Force collaboration-engine version to 3.2.1

* Updated generated files

### CODE REVIEW

File: frontend/generated/theme-ce-demo.generated.js [hunk 1]
```diff
@@ -35,7 +35,13 @@ const createLinkReferences = (css, target) => {
 
 // target: Document | ShadowRoot
 export const injectGlobalCss = (css, target, first) => {
-
```
Comment (Legioth) [inline frontend/generated/theme-ce-demo.generated.js:38]: Shouldn't this file be in `.gitignore`?

File: pom.xml [hunk 2]
```diff
@@ -95,6 +95,11 @@
             <artifactId>a-vaadin-helper</artifactId>
             <version>1.6.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>collaboration-engine</artifactId>
+            <version>3.2.1</version>
```
Comment (Legioth) [inline pom.xml:101]: Could maybe have a comment in the code about why this is overridden?

Comment (tulioag) [reply pom.xml:104]: Done

File: frontend/generated/theme-ce-demo.generated.js (hunk 1 above)
Comment (tulioag) [reply frontend/generated/theme-ce-demo.generated.js:38]: Done

**Comment by Legioth** [comment]
Seems like I was accidentally commenting on the wrong PR. Those changes that I suggested should go to `master` first rather than being applied directly to `production`.

**Comment by tulioag** [comment]
Reverted. PRs created for master.

**Comment by Legioth** [comment]
Would it be clearer to instead recreate the merge instead of having reverts in it?

**Comment by tulioag** [comment]
Recreated the branch.

**Review by Legioth — CHANGES_REQUESTED** [review]
https://collaboration-engine-stg.demo.vaadin.com/ doesn't seem to allow opening the edit dialog again after it's been closed.

**Comment by tulioag** [comment]
Outdated.

**Pull request closed without merging**